ddp-workspace/projects/ddp-pancan/src/app/components/faq-section/faq-section.component.scss

```diff
@@ -0,0 +1,52 @@
+$border-color: #EBEBEB;
```

Comment:
should we ask Erin to provide the app palette? (you can find it in theme.scss in each study)

Reply:
It would be nice. 
I have asked Erin for that.

Reply:
do we know when approximately it will be provided?

Reply:
I don't have the answer.